Document the allow_audit_log_prefixing flag (#702)
hashicorp/web-unified-docs: content/vault/v1.18.x/content/docs/configuration/index.mdx

  [os.TempDir()](https://pkg.go.dev/os#TempDir) (usually `/tmp` on Unix systems). This can be updated on a running Vault process
  with a SIGHUP signal.

> - `allow_audit_log_prefixing` `(bool: false)` - Enable to allow a constant prefix to be added to audit log entries on 
>   [file sinks](/vault/docs/audit/file).

Reviewer: ```suggestion
- `allow_audit_log_prefixing` `(bool: false)` - Enable to allow prefixes on audit log entries for
  [file sinks](/vault/docs/audit/file).
```
Edited to match previous suggestion
